src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/HarmonicSchwarzschild.hpp

```diff
@@ -0,0 +1,935 @@
+ * \f}
+ *
+ * The lapse and its derivatives are
+ *
+ * \f{align}
+ *     \alpha &= \gamma_{rr}^{-1/2},\\
+ *     \partial_t \alpha &= 0,\\
+ *     \partial_i \alpha &=
+ *         -\frac{1}{2} \gamma_{rr}^{-3/2} \partial_r \gamma_{rr} \frac{X_i}{r}
+ * \f}
+ *
+ * The shift and its time derivative are
+ *
+ * \f{align}
+ *     \beta^i &= \frac{2 M}{M+r} \frac{X^i}{r} \frac{1}{\gamma_{rr}},\\
+ *     \partial_t \beta^i &= 0
+ * \f}
+ *
+ * The spatial derivative of the shift is computed in terms of the following
+ * quantities:
+ *
+ * \f{align}
+ *     f_3 &= \frac{1}{r} \frac{1}{\gamma_{rr}} \left(\frac{2M}{M+r}\right)^2,\\
+ *     f_4 &=
+ *         -f_3 -
+ *         \frac{1}{M} \frac{1}{\gamma_{rr}}
+ *             \left(\frac{2M}{M+r}\right)^3 -
+ *         \partial_r \gamma_{rr} \left(\frac{2 M}{M+r}
+ *             \frac{1}{\gamma_{rr}}\right)^2
+ * \f}
+ *
+ * In terms of these, the shift's spatial derivative is
+ *
+ * \f{align}
+ *     \partial_k \beta^i &= f_4 \frac{X^i}{r} \frac{X_k}{r} + \delta_k^i f_3
+ * \f}
+ */
+class HarmonicSchwarzschild : public AnalyticSolution<3_st>,
+                              public MarkAsAnalyticSolution {
+ public:
+  struct Mass {
+    using type = double;
+    static constexpr Options::String help = {"Mass of the black hole"};
+    static type lower_bound() { return 0.; }
+  };
+  struct Center {
+    using type = std::array<double, volume_dim>;
+    static constexpr Options::String help = {
+        "The [x,y,z] center of the black hole"};
+  };
+  using options = tmpl::list<Mass, Center>;
+  static constexpr Options::String help{
+      "Schwarzschild black hole in Cartesian coordinates with harmonic gauge"};
+
+  HarmonicSchwarzschild(double mass, Center::type center,
+                        const Options::Context& context = {});
+
+  explicit HarmonicSchwarzschild(CkMigrateMessage* /*unused*/) {}
+
+  HarmonicSchwarzschild() = default;
+  HarmonicSchwarzschild(const HarmonicSchwarzschild& /*rhs*/) = default;
+  HarmonicSchwarzschild& operator=(const HarmonicSchwarzschild& /*rhs*/) =
+      default;
+  HarmonicSchwarzschild(HarmonicSchwarzschild&& /*rhs*/) = default;
+  HarmonicSchwarzschild& operator=(HarmonicSchwarzschild&& /*rhs*/) = default;
+  ~HarmonicSchwarzschild() = default;
+
+  /*!
+   * \brief Computes and returns spacetime quantities for a Schwarzschild black
+   * hole in Cartesian coordinates with harmonic gauge at a specific position
+   * relative to the center of the black hole
+   *
+   * \details Quantities returned are specified by `gr::AnalyticSolution::tags`.
+   * The position relative to the center of the black hole is defined
+   * as \f$X^i = \left(x^i - C^i\right)\f$, where \f$C^i\f$ is the Cartesian
+   * coordinates of the center of the black hole and \f$x^i\f$ is the
+   * Cartesian coordinates of the point where we're wanting to compute the
+   * spacetime quantities
+   *
+   * \param x the position relative to the center of the black hole
```

Comment:
Just to confirm, the parameter x passed into these functions is x^i - c^I, not x^i? I would have thought these functions just take x^I, but this line in the documentation makes me think otherwise. Does Kerr Schild work the same way?

Reply:
Whoops, I think this is a documentation mistake on my part. It  should just be x^i, not X^i = x^i - C^i. I'll fix the documentation